thunder/tests/test_dynamo.py

```diff
@@ -988,8 +989,10 @@ def check(file_name, cmd):
     cmd = [sys.executable]
     if use_pytest_benchmark:
         cmd = cmd + ["-m", "pytest"]
-    for fname in [s1, s2, s3]:
-        check(fname, cmd)
+
+    # Randomly pick one file to run to save time
+    fname = random.choice([s1, s2, s3])
```

Comment:
If you want to introduce sampling like this then we should use the `hypothesis` package (https://hypothesis.readthedocs.io/en/latest/) so that the random choice is visible and easy to reproduce